instance/models/instance.py

```diff
@@ -586,4 +669,19 @@ def provision(self):
 
         return (server, ansible_log)
 
+    def provision_external_databases(self):
+        """
+        Provision external databases for this instance
+        """
+        if settings.INSTANCE_MYSQL_URL_OBJ:
+            connection = mysql.connect(
+                host=settings.INSTANCE_MYSQL_URL_OBJ.hostname,
+                user=settings.INSTANCE_MYSQL_URL_OBJ.username,
+                passwd=settings.INSTANCE_MYSQL_URL_OBJ.password or '',
+                port=settings.INSTANCE_MYSQL_URL_OBJ.port or 3306,
+            )
+            cursor = connection.cursor()
+            database_name = connection.escape_string(self.database_name).decode()
+            cursor.execute('CREATE DATABASE IF NOT EXISTS {0}'.format(database_name))
```

Comment:
I actually tried that first, it doesn't work for database names. The driver escapes it incorrectly. I will add a comment explaining this